spec/namespaces/ml.yaml

```diff
@@ -102,6 +122,15 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ml.search_models@200'
+    post:
```

Comment:
we support get and post both. [Ref](https://quip-amazon.com/VhHOAptBw9u0/Resource-clean-up-on-OASIS-OASIS-Cell-Routing#temp:C:ZBW58896dac31dd4135baa7263fa)

Reply:
Not a part of this PR, but I noticed in the line 225 of file, model_format is marked as required. model_format is required only if the model is not a remote model. [Ref](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java#L136-L149)

Reply:
Got it, I will change that.

Reply:
> we support get and post both. [Ref](https://quip-amazon.com/VhHOAptBw9u0/Resource-clean-up-on-OASIS-OASIS-Cell-Routing#temp:C:ZBW58896dac31dd4135baa7263fa)

Yes, get is already added on this file, on line 116.